samples/drivers/mbox/boards/nrf54h20pdk_nrf54h20_cpuapp_bellboard.overlay

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+mbox: &cpuapp_bellboard {
+	interrupts = <96 NRF_DEFAULT_IRQ_PRIORITY>;
+	interrupt-names = "bellboard0";
+	status = "okay";
+};
```

Comment:
These might be worth moving to the default board configuration, sans label.

Reply:
Could you suggest a default IRQ configuration?

Reply:
Let's come back to it later.